e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -330,6 +331,31 @@ describe("issue 12720", () => {
     clickThrough("12720_SQL");
     clickThrough("Orders");
   });
+
+  it("should apply yhe specific (before|after) filter on a native question with field filter (metabase#47172)", () => {
+    visitDashboard(ORDERS_DASHBOARD_ID);
+
+    getDashboardCard(1).within(() => {
+      cy.findByTestId("TableFooter").should("exist");
+      cy.findByText("There was a problem displaying this chart.").should(
+        "not.exist",
+      );
+
+      cy.log("Drill down to the question");
+      cy.intercept("POST", "/api/card/*/query").as("cardQuery");
+      cy.findByText(questionDetails.name).click();
+    });
+
+    cy.location("search").should("eq", `?filter=${dashboardFilter.default}`);
+    cy.wait("@cardQuery").then(({ response }) => {
+      expect(response.body.status).to.eq("completed");
+    });
```

Comment:
Do we really need this assertion?

```suggestion
    cy.wait("@cardQuery");
```